modules/xfem/test/tests/bimaterials/tests

```diff
@@ -7,6 +7,12 @@
     use_old_floor = true
     map = false
     unique_id = true
+    requirement = 'The XFEM module shall have the capability to calculate '
+                  'displacement in two glued, layered materials with the '
+                  'interface location and global stress determined by a '
+                  'level set function.'
```

Comment:
displacement->mechanical response

Is the global stress determined by the level set function? That doesn't seem right. Same with the case below.